ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbarManager.java

```diff
@@ -84,6 +84,14 @@ public void setTitle(ReactToolbar view, @Nullable String title) {
     view.setTitle(title);
   }
 
+  @ReactProp(name = "rtl")
+  public void setLayoutDirection(ReactToolbar view, @Nullable Boolean flag) {
+    if (flag.equals(true)) {
```

Comment:
If `flag` is `@Nullable`, then won't `flag.equals(true)` throw a `NullPointerException`?